metabase/metabase
# Add `sort_query` tool

How to verify:
- Open a question or a table
- New -> Metabot request -> Sort by {{column name}}

Feels hacky; unclear how this should be actually implemented. For now just want to go with this and make more tools and reconsider the approach later.
